lib/rex/proto/kerberos/model/transited_encoding.rb

```diff
@@ -0,0 +1,111 @@
+# -*- coding: binary -*-
+
+module Rex::Proto::Kerberos::Model
+  # This class provides a representation of a Kerberos ticket that helps
+  # a client authenticate to a service.
+  class TransitedEncoding < Element
+
+    attr_accessor :tr_type  # [0] Int32 -- must be registered --,
```

Comment:
Could we add real yard docs here, so that we get nicer code completion. I'm not sure how yard will handle this string being at the end 👀

Reply:
I think I'm googling wrong but I can't see examples of how you're supposed to yard doc attributes? I can see this https://www.rubydoc.info/gems/yard/file/docs/Tags.md#attr:~:text=%40return-,Directive%20List,-%40!attribute%20%5Br%20%7C%20w
but that seems to be for defining it not just documenting it?

Reply:
https://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md#documenting-attributes

Although most folk have copy/pasta'd the `@!attribute` approach, so I've been using that for consistency